jobwatch/jobwatch.py

```diff
@@ -174,16 +173,19 @@ def set_report_attrs(jobwatches):
         if jw.stale:
             jw.age_str = '<span style="color:red";>{}</span>'.format(
                 jw.age_str)
-        if i_jw == 0 or jw.type != jobwatches[i_jw - 1].type:
-            jw.span_cols_text = jw.type
+
+        this_type = getattr(jw, 'type', 'Job')
+        last_type = getattr(jobwatches[i_jw -1], 'type', 'Job')
+        if i_jw == 0 or this_type != last_type:
+            jw.span_cols_text = this_type
```

Comment:
I'm not sure if jw type should be a property, but this more explicit handling of the attr at least seemed not to break much (related to comment at https://github.com/sot/jobwatch/pull/41#discussion_r551679164) .